### PR TITLE
AP_HAL_F4light: fixed compilation for boards AirbotF4, AirbotV2 and CL_Racing_F4.

### PR DESCRIPTION
**Pull request by night-ghost**
Corrected prefix for binaries too

AP_Notify: added personal support for AP_HAL_F4light

### CODE REVIEW

File: libraries/AP_Notify/AP_Notify.cpp [hunk 1]
```diff
@@ -221,15 +221,20 @@ void AP_Notify::add_backends(void)
...
+# endif
     ADD_BACKEND(new ToshibaLED_I2C(TOSHIBA_LED_I2C_BUS_EXTERNAL));
     ADD_BACKEND(new ToshibaLED_I2C(TOSHIBA_LED_I2C_BUS_INTERNAL));
     ADD_BACKEND(new Display());
-#lese
+#elif CONFIG_HAL_BOARD == HAL_BOARD_F4LIGHT
+    ADD_BACKEND(new ToshibaLED_I2C(TOSHIBA_LED_I2C_BUS_EXTERNAL));
+    ADD_BACKEND(new Display());
+    ADD_BACKEND(new Buzzer());
+//    ADD_BACKEND(new AP_BoardLED2()); // needs AP_BoardLED2 in master
```
Comment (rmackay9) [inline libraries/AP_Notify/AP_Notify.cpp]: normally we don't leave commented out code in there so probably best to remove this line.

**Comment by rmackay9** [comment]
This looks OK to me.  I haven't reviewed every line but it seems to be mostly renaming of files.  Personally I probably would have split up the 2nd commit a bit (i.e. the wayback-mode removal and build change) but it's not the end of the world.

File: libraries/AP_Notify/AP_Notify.cpp (hunk 1 above)
Comment (night-ghost) [reply libraries/AP_Notify/AP_Notify.cpp]: this commented code waits for #7585, so after it will be merged this becomes just a reminder

**Comment by night-ghost** [comment]
>the wayback-mode removal

this not was a wayback-mode but just a testing code, which is an absolutely garbage in HAL code

**Comment by tridge** [comment]
merged, thanks!

**Pull request closed without merging**